ui/page/livestreamSetup/view.jsx

```diff
@@ -161,237 +177,277 @@ export default function LivestreamSetupPage(props: Props) {
     return (
       <div className={'w-full flex items-center justify-between'}>
         <span>{title}</span>
-        {!hideBtn && (
+        {!hideBtn && !isMobile && (
           <Button
             button="primary"
             iconRight={ICONS.ADD}
-            onClick={() => doNewLivestream(`/$/${PAGES.UPLOAD}?type=${PUBLISH_MODES.LIVESTREAM.toLowerCase()}`)}
+            onClick={() => createNewLivestream()}
             label={__('Create or Schedule a New Stream')}
           />
         )}
       </div>
     );
   };
 
-  return (
-    <Page>
-      {/* channel selector */}
-      {!fetchingChannels && (
-        <>
-          <div className="section__actions--between">
-            <ChannelSelector hideAnon />
-          </div>
-        </>
-      )}
+  const [tab, setTab] = React.useState('Publish');
 
-      {/* livestreaming disabled */}
-      {!livestreamEnabled && (
-        <div style={{ marginTop: '11px' }}>
-          <h2 style={{ marginBottom: '15px' }}>
-            {__('This account has livestreaming disabled, please reach out to hello@odysee.com for assistance.')}
-          </h2>
+  const HeaderMenu = (e) => {
+    return (
+      <>
+        <Button
+          key={'Publish'}
+          iconSize={18}
+          label={'Publish'}
+          button="alt"
+          onClick={() => {
+            // $FlowFixMe
+            setTab('Publish');
+          }}
+          disabled={e.disabled}
+          className={classnames('button-toggle', { 'button-toggle--active': tab === 'Publish' })}
+        />
+        <Button
+          key={'Setup'}
+          iconSize={18}
+          label={'Local Setup'}
+          button="alt"
+          onClick={() => {
+            // $FlowFixMe
+            setTab('Setup');
+          }}
+          disabled={e.disabled || e.isEditing}
+          className={classnames('button-toggle', { 'button-toggle--active': tab === 'Setup' })}
+        />
+      </>
+    );
+  };
+
+  return (
+    <Page className="uploadPage-wrapper">
+      {balance < 0.01 && <YrblWalletEmpty />}
+      {balance >= 0.01 && fetchingChannels && (
+        <div className="main--empty">
+          <Spinner />
         </div>
       )}
+      <h1 className="page__title">
+        <Icon icon={ICONS.VIDEO} />
+        <label>
+          {__('Go Live')}
+          {!isClear && <Button onClick={() => clearPublish()} icon={ICONS.REFRESH} button="primary" label="Clear" />}
+        </label>
+      </h1>
+      <HeaderMenu disabled={balance < 0.01} isEditing={editingURI} />
 
-      {/* show livestreaming frontend */}
-      {livestreamEnabled && (
-        <div className="card-stack">
-          {/* getting channel data */}
-          {fetchingChannels && (
-            <div className="main--empty">
-              <Spinner delayed />
+      {tab === 'Setup' && (
+        <div className="disabled">
```

Comment:
I think this is causing "Local Setup" to always be grayed out

<img width="572" alt="image" src="https://user-images.githubusercontent.com/64950861/177071538-14758944-1dc5-4dcb-b7c0-179df7b1ee8a.png">